### PR TITLE
Add modified scratch storage package as dependency

### PR DESCRIPTION
**Pull request by taybenlor**
Just throwing this in so I can refer to this package from scratch-iframe

### CODE REVIEW

**Review by wryun — APPROVED** [review]

File: src/lib/project-saver-hoc.jsx [hunk 1]
```diff
@@ -138,14 +138,18 @@ const ProjectSaverHOC = function (WrappedComponent) {
             }
         }
         scheduleAutoSave () {
-            if (this.props.isShowingSaveable && this.props.autoSaveTimeoutId === null) {
+            if (this.props.autoSaveTimeoutId === null) {
+            // Grok wants autosave to happen regardless of whether saving is enabled
+            // if (this.props.isShowingSaveable && this.props.autoSaveTimeoutId === null) {
```
Comment (wryun) [inline src/lib/project-saver-hoc.jsx]: Maybe some explanation here? What does it mean to save an unsaveable project?

**Comment by taybenlor** [comment]
@wryun I removed some of that "no saving" stuff because it turns out it's unnecessary now that I've hidden the menubar.

I added some other stuff because I noticed if I rapidly change projects then autosaving breaks.

Ready for review

File: package.json [hunk 2]
```diff
@@ -108,11 +109,11 @@
     "redux-throttle": "0.1.1",
     "rimraf": "^2.6.1",
     "scratch-audio": "0.1.0-prerelease.20190925183642",
-    "scratch-l10n": "3.6.20191210224141",
     "scratch-blocks": "0.1.0-prerelease.1574180945",
+    "scratch-l10n": "3.6.20191210224141",
     "scratch-paint": "0.2.0-prerelease.20191217213717",
     "scratch-render": "0.1.0-prerelease.20191217212645",
-    "scratch-storage": "1.3.2",
+    "scratch-storage": "git://github.com/groklearning/scratch-storage.git#grok-release-17-01-2020",
```
Comment (wryun) [inline package.json:116]: hang on, did you just tag a release off an unmerged branch?

File: src/components/gui/gui.jsx [hunk 3]
```diff
@@ -199,7 +199,9 @@ const GUIComponent = props => {
                         onRequestClose={onRequestCloseBackdropLibrary}
                     />
                 ) : null}
-                <MenuBar
+                {/* Grok: We want the menubar hidden for our interface
```
Comment (wryun) [inline src/components/gui/gui.jsx:202]: not as advertised on the tin :)

I mean, it's nice-ish to do this, though we could hide it in the CSS in scratch-iframe instead if we're being lazy (and then we carry fewer random hacks in this repo)... choose your poison?

**Review by wryun — APPROVED** [review]
LGTM :ghost: (with minor caveats as mentioned)

**Review by wryun — APPROVED** [review]
Still LGTM (aside from earlier comments).

**Pull request merged**